less/items.less

```diff
@@ -251,7 +251,7 @@
         .configuration-mode-control {
           flex: 1 0;
           margin-inline-start: 0.5em;
-          a { text-align: inline-start; }
+          a { text-align: start; }
```

Comment:
I think we generally use `text-align: left` elsewhere in the code, if consistency is desired.

Reply:
I've been generally trying to use logical properties and values wherever possible over physical to allow for potential future support of other reading directions.